src/util.js

```diff
@@ -695,3 +695,44 @@ export const normaliseExtraIdentity = (extraIdentity) => {
     ]
   }, []).join('_')
 }
+
+
+/**
+ * Checks if at least one of the provided `permissions` is privileged enough to access the `route`
+ * with the given `method`.
+ */
+export const validateUserAccess = ({
```

Comment:
as this function is returning a bool value, how about a more expressive name like `hasUserAccess`?